code/modules/mob/living/carbon/give.dm

```diff
@@ -1,47 +1,58 @@
 /mob/living/carbon/verb/give()
 	set category = "IC"
 	set name = "Give"
-	set src in oview(1)
```

Comment:
if i'm not wrong this makes so people nearby can use the verb, make sure this still keeps happening, i know verb set stuff is weird but thats how it is